Rakefile

```diff
@@ -30,6 +30,14 @@ begin
     title 'Building the gem'
   end
 
+  task :check_submodules do
+    title 'Ensuring submodules are initialized'
+    submodule_help_msg = 'Run git submodule update --init --recursive to ensure submodules are initialized'
+    raise submodule_help_msg if `git submodule status`.split("\n").any? do |line|
```

Comment:
Changing this to `error(submodule_help_msg) if…` raises an exception:
```
NoMethodError: undefined method `error' for main:Object
```
Other tasks appear to use `raise` for this purpose as well. Maybe I misinterpreted?